src/Actions/DeleteAppFolderAction.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Actions;
+
+use Illuminate\Filesystem\Filesystem;
+
+class DeleteAppFolderAction
```

Comment:
Maybe a more general delete action would be better. As we maybe need to delete files some day.